### PR TITLE
Buyer's Guide og tags & main nav share icons

### PR DESCRIPTION
**Pull request by mmmavis**
Related #1899

https://foundation-mofostaging-pr-1926.herokuapp.com/privacynotincluded/

### Only covers the following
- Got share icons on main nav to work
  - Note: I got share icons to work without using ShareProgress. I found out [Facebook stopped supporting prefilled text for share](https://developers.facebook.com/docs/apps/review/prefill) anyway so I guess we are good leaving ShareProgress out. 
- Added OG tags

### Not covered in this PR
- creep-o-meter related share

---

# ❗️ Remember to revert changes in `network-api/networkapi/buyersguide/views.py` before merging

### CODE REVIEW

**Comment by kristinashu** [comment]
Yay looks great!

Copy link doesn't work on my iPhone but that is an issue on the live site and also on the category page so we can try to sort that out later.

**Comment by alanmoo** [comment]
Let's actually go ahead and leave the removal of the login prevention when we merge this since we want this to be unlocked today anyway.

File: network-api/networkapi/buyersguide/views.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def get_average_creepiness(product):
 
 # Login required so we can continue to develop this and merge into master without the public seeing it.
 # Remove this when we're ready to launch.
-@login_required
+# @login_required
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/views.py]:
```suggestion
@login_required
```

File: network-api/networkapi/buyersguide/views.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def buyersguide_home(request):
     })
 
 
-@login_required
+# @login_required
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/views.py]:
```suggestion
@login_required
```

File: network-api/networkapi/buyersguide/views.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ def category_view(request, categoryname):
...
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/views.py]:
```suggestion
 @login_required
```

File: network-api/networkapi/buyersguide/views.py [hunk 4]
```diff
@@ -64,7 +64,7 @@ def product_view(request, productname):
...
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/views.py]:
```suggestion
@login_required
```

**Pull request merged**